### DISCUSSION TITLE
Example in domain_templates: computer_vision_fine_tuning

### DISCUSSION THREAD

**Discussion by amisev** [Lightning-AI/pytorch-lightning#7081] [code help: CV] [answered]
I found that in the code below sigmoid activation applied before binary_cross_entropy_with_logits loss:
https://github.com/PyTorchLightning/pytorch-lightning/blob/71b4611c64059d7589e4d80115209fd2c89e8bdb/pl_examples/domain_templates/computer_vision_fine_tuning.py#L196-L231
From the documentation https://pytorch.org/docs/stable/generated/torch.nn.BCEWithLogitsLoss.html#torch.nn.BCEWithLogitsLoss:
```This loss combines a Sigmoid layer and the BCELoss in one single class. ```
Is that performed intentionally? Or it's just a bug?

**Comment by awaelchli** [comment, marked as answer]
Yes, the sigmoid should be removed from the forward, because BCEWithLogits already contains the sigmoid. 
Do you want to send a PR to update the example?

**Reply by amisev** [reply]
I've created [PR](https://github.com/PyTorchLightning/pytorch-lightning/pull/7096).
I replaced bce_loss_with_logits with bce_loss. Perhaps, It'll be more convenient to work with predicted probabilities rather than some scores.
